docs/user/cropping-and-transforming.md

```diff
@@ -5,28 +5,29 @@
 > away can still be restored.
 
 ```python
-from pypdf import PdfWriter, PdfReader
+from pypdf import PdfReader, PdfWriter
 
 reader = PdfReader("example.pdf")
 writer = PdfWriter()
 
-# add page 1 from reader to output document, unchanged:
+# Add page 1 from reader to output document, unchanged
```

Comment:
I would probably add the final period to the lines instead to mark it as a sentence to be consistent with the further explanations.

Reply:
Done.